### PR TITLE
feat: AuthorizationCredential Context

### LINKED ISSUE #18: AuthorizationCredential Context
https://github.com/trustbloc/edge-adapter/issues/188

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #18 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: vc/authorization-credential-v1.jsonld [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      },
+      "issuerDIDDoc":{
+         "@id":"ex:issuerDIDDoc",
+         "@type":"DIDDoc"
+      },
+      "rpDIDDoc":{
+         "@id":"ex:rpDIDDoc",
+         "@type":"DIDDoc"
+      },
+      "userDID":{
```
Comment (troyronda) [inline vc/authorization-credential-v1.jsonld:31]: we should also rename user.

Comment (llorllale) [reply vc/authorization-credential-v1.jsonld:31]: @troyronda this is just the user's DID

Comment (rolsonquadras) [reply vc/authorization-credential-v1.jsonld:31]: @troyronda any specific reason to rename ? Do you see that the value of this attribute could be `non-did` ?

Comment (troyronda) [reply vc/authorization-credential-v1.jsonld:31]: my hesitation is not the DID aspect but the “user” term :)

Comment (rolsonquadras) [reply vc/authorization-credential-v1.jsonld:31]: Oh :) any suggestions ?

Comment (troyronda) [reply vc/authorization-credential-v1.jsonld:31]: subject would be the more generic form.

Comment (rolsonquadras) [reply vc/authorization-credential-v1.jsonld]: [Updated](https://github.com/trustbloc/context/compare/b65a651f9b159da815616be3a65de5816be439b6..2faf6e8a0a9d9c3347e05b2d8ebcd43e7c315660).

**Pull request merged**